File: 2_POO vs Estruturada_alunos/2_comparacao-php_alunos/varios_estruturada.php
<?php
// ========================================== 
// PARTE 1: PROGRAMAÇÃO ESTRUTURADA
// ==========================================

// Funções para manipular os dados


// 1ª Digitação (Aqui)


// Criando vários cachorros
$cachorros = [
    ["nome" => "Nelson", "comida" => 3, "sono" => false],
    ["nome" => "Jeremias", "comida" => 1, "sono" => true],
    ["nome" => "Cléo", "comida" => 247, "sono" => true],
    ["nome" => "Francisco", "comida" => 5, "sono" => true],
    ["nome" => "Conan", "comida" => 19, "sono" => false],
    ["nome" => "Leleco", "comida" => 3, "sono" => false],
    ["nome" => "Babi", "comida" => 13, "sono" => true]
];

// Manipulando os cachorros


// 2ª Digitação (Aqui)


// Exibindo os resultados no navegador
echo "<!DOCTYPE html>";
echo "<html lang='pt-BR'>";
echo "<head>";
echo "    <meta charset='UTF-8'>";
echo "    <meta name='viewport' content='width=device-width, initial-scale=1.0'>";
echo "    <title>Resultados dos Cachorros</title>";
echo "</head>";
echo "<body>";
echo "    <h1>Resultados dos Cachorros</h1>";
echo "    <ul>";

foreach ($cachorros as $cachorro) {
    echo "        <li><strong>" . $cachorro["nome"] . "</strong> agora tem <strong>" . $cachorro["comida"] . "</strong> unidades de comida e está com sono? <strong>" . ($cachorro["sono"] ? 'Sim' : 'Não') . "</strong></li>";
}

echo "    </ul>";
echo "</body>";
echo "</html>";

File: 2_POO vs Estruturada_alunos/Python/varios_estruturada.py
# Funções para manipular os dados
def comer(quantidade_comida):
    return max(quantidade_comida - 1, 0)

def dormir():
    return True

# Criando vários cachorros
cachorros = [
    {"nome": "Nelson", "comida": 3, "sono": False},
    {"nome": "Jeremias", "comida": 1, "sono": True},
    {"nome": "Cléo", "comida": 247, "sono": True},
    {"nome": "Francisco", "comida": 5, "sono": True},
    {"nome": "Conan", "comida": 19, "sono": False},
    {"nome": "Leleco", "comida": 3, "sono": False},
    {"nome": "Babi", "comida": 13, "sono": True}
]

# Manipulando os cachorros
for cachorro in cachorros:
    cachorro["comida"] = comer(cachorro["comida"])

# Exibindo os resultados no terminal
for cachorro in cachorros:
    print(f"{cachorro['nome']} agora tem {cachorro['comida']} unidades de comida.")
    print(f"{cachorro['nome']} está com sono? {'Sim' if cachorro['sono'] else 'Não'}")
